frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -15,8 +19,60 @@
     ['motherTongue', ['English', 'Suomi', 'Svenska']]
   ]);
 
-  const labelClass = 'label-sm label mx-6 my-2 text-secondary';
-  const disclaimerClass = 'm-0 p-0 text-sm text-secondary';
+  const labelClass = 'w-6/12 label-sm label mx-6 my-2 text-secondary';
+  const disclaimerClass = 'mx-6 my-0 p-0 text-sm text-secondary';
+  const headerClass = 'mx-6 my-0 p-0 text-m text-secondary';
+  const inputClass =
+    'input-ghost input input-sm w-full pr-2 text-right disabled:border-none disabled:bg-base-100';
+
+  let portraitInput: HTMLElement | null;
+  let portraitLabel: HTMLElement | null;
+
+  // function for clicking the portrait input field with space
+  const handlePortraitInput = (event: KeyboardEvent) => {
+    if (event.code === 'Space') {
+      event.preventDefault(); // Prevent default behavior (e.g., scrolling the page)
+      portraitInput?.click();
+    }
+  };
```

Comment:
I think the logic for allowing the portrait input to be selected using space bloats this components an unnecessary amount. I suggest we make a separate component for the file input that also handles the keyboard input.

Reply:
Let's leave it as it is for now, I'll leave a TODO if we want to reafactor in the future.